WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -98,6 +100,8 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 				foreach (var finishedCoinJoin in finishedCoinJoins)
 				{
 					var logPrefix = $"Wallet: `{finishedCoinJoin.Wallet.WalletName}` - Coinjoin client";
+					var freezeCoins = true;
```

Comment:
this is unnecesary, just freeze them if success is true and you can remove the flag from everywhere.

Reply:
ACK